test/setup/platforms/openshift4.ts

```diff
@@ -49,9 +49,35 @@ export async function exportKubeConfig(): Promise<void> {
 }
 
 export async function clean(): Promise<void> {
+  // Kubernetes will be stuck trying to delete these resources if we don't clear the finalizers.
   await Promise.all([
-    kubectl.deleteNamespace('services'),
-    kubectl.deleteNamespace('snyk-monitor'),
+    kubectl.patchResourceFinalizers('customresourcedefinition', 'snykmonitors.charts.helm.k8s.io', 'snyk-monitor').catch((error) => {
+      console.log(error);
+    }),
+    kubectl.patchResourceFinalizers('snykmonitors.charts.helm.k8s.io', 'snyk-monitor', 'snyk-monitor').catch((error) => {
+      console.log(error);
+    }),
+  ]);
+
+  await Promise.all([
+    kubectl.deleteResource('customresourcedefinition', 'snykmonitors.charts.helm.k8s.io', 'default').catch((error) => {
```

Comment:
I really don't like it, but I don't see a better alternative 😆

Reply:
How come? 😄 The Operator creates these files, we don't have a good way to clean them up...

Reply:
I mean that I don't like the `Promise.all([blabla.catch(err)])`

Reply:
Ah, fair enough... I find it cleaner than try-catching every individual promise... Plus the order doesn't matter (as long as they are grouped like that, so that `patch`-ing comes first)